File: app/src/main/java/com/kota/Bahamut/ListPage/TelnetListPageItem.java
package com.kota.Bahamut.ListPage;

public class TelnetListPageItem {
    public int Number = 0;
    public boolean isBlocked = false;
    public boolean isDeleted = false;
    public boolean isLoading = false;

    public void set(TelnetListPageItem aData) {
        if (aData != null) {
            this.isDeleted = aData.isDeleted;
            this.isLoading = aData.isLoading;
        }
    }

    public void clear() {
        this.Number = 0;
        this.isDeleted = false;
        this.isLoading = false;
        this.isBlocked = false;
    }
}

File: app/src/main/java/com/kota/Bahamut/listPage/TelnetListPageItem.kt
package com.kota.Bahamut.listPage

open class TelnetListPageItem {
    @JvmField
    var itemNumber: Int = 0
    @JvmField
    var isBlocked: Boolean = false
    @JvmField
    var isDeleted: Boolean = false
    var isLoading: Boolean = false

    open fun set(aData: TelnetListPageItem?) {
        if (aData != null) {
            this.isDeleted = aData.isDeleted
            this.isLoading = aData.isLoading
        }
    }

    open fun clear() {
        this.itemNumber = 0
        this.isDeleted = false
        this.isLoading = false
        this.isBlocked = false
    }
}
